.env

```diff
@@ -0,0 +1,6 @@
+NODE_ENV=development
```

Comment:
.env file shouldnt be tracked. please remove it from git.
if you want to store default values - create `.env.development` and so on. Next time new dev will copy its content to .env file locally